src/Driver/driver_configs.jl

```diff
@@ -17,13 +17,27 @@ struct AtmosLESSpecificInfo <: ConfigSpecificInfo end
 struct AtmosGCMSpecificInfo{FT} <: ConfigSpecificInfo
     domain_height::FT
     nelem_vert::Int
-    nelem_horz::Int
+    nelem_horiz::Int
```

Comment:
❤️. The previous name bothered me quite a bit

Reply:
haha I am glad I was not the only one ;)